### ISSUE TITLE
The concept of homotopy type

### ISSUE THREAD

**Issue by marcbezem** [UniMath/SymmetryBook#185]
Should be clarified somewhere in the book, currently Appendix B.3.

**Comment by DanGrayson** [comment]
Or we should not use it.  The use most people in our field put it to conflicts with the classical definition, and equivalence class of spaces modulo homotopy equivalence.

**Comment by bidundas** [comment]
I agree that ”homotopy type” should have as small a place in the text as possible.  

A thing I liked about univalence is that it made my colloquialism ”up to deformation” (which I’ve always favored when giving popularizations) into something precise.

Bjorn
